CodingGuidelines.md

```diff
@@ -100,21 +101,46 @@ private Foo()
 }
  ```
 
+## Spacing
+
+Please be sure not to add additional spaces between square brackets and parenthesis:
```

Comment:
I'm not sure I intended my PR review to mean that we should add this to the guidelines.

I've never created coding standards for a shared repo before, but this seems like a nitpicky thing to enforce. Perhaps it's necessary though, and I'd appreciate others weighing in.

Reply:
Well it's part of MSDN as as we clearly say follow MSDN when it's not explicitly called out, I decided add it to make it clear.

Reply:
We could also enforce this by adding a vs configuration file, but I feel that's a bit extreme for editor default settings.

Reply:
Do you have a link to the MSDN guideline related to this?

Reply:
[Sure](https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/inside-a-program/coding-conventions#layout-conventions)

> Use the default Code Editor settings

Reply:
Now let's get back to reviewing some important PRs like gestures. We really need to get that in sooner rather than later.

Reply:
"Use the default Code Editor settings" doesn't say anything explicitly about this rule, but okay.

>Now let's get back to reviewing some important PRs like gestures

Larger PRs require more targeted time to review.

Additionally, I have an outstanding change request from before you most recently "bump"ed that PR...

Reply:
Please reword the header to 
"Do not add additional spaces between square brackets and parenthesis:"
Old statement was too passive